modules/internal/ChapelDistribution.chpl

```diff
@@ -743,6 +744,29 @@ module ChapelDistribution {
       return (ret_arr, ret_dom);
     }
 
+    proc chpl_setResizePolicy(policy: chpl_DdataResizePolicy) {
+      _resizePolicy = policy;
+    }
+
+    proc chpl_isElementTypeDefaultInitializable(): bool {
```

Comment:
I think this and the following should return `param`.

While I say that, I am also remembering an issue about arrays with runtime-typed elements. ie, array of arrays/domains. Have you bumped into something like that and that's why these aren't returning `param`s?

Reply:
Just tested and it's as I suspected - these can't be param because dynamic dispatch needs to happen in the manager code due to the receiver only being of type `BaseArr`. I'm pretty sure `param` methods just get selected based on the static type of the receiver.